packages/cli-kit/src/session.ts

```diff
@@ -215,6 +217,9 @@ ${token.json(applications)}
     } catch (error) {
       if (error instanceof InvalidGrantError) {
         newSession = await executeCompleteFlow(applications, fqdn)
+      } else if (error instanceof InvalidRequestError) {
+        await secureStore.remove()
+        throw new AbortError('\nError validating auth session', "We've cleared the current session, please try again")
```

Comment:
I've tried to touch as little as possible, but shouldn't we just execute the complete flow for these two cases? Why removing the session and asking the user to try again instead of just doing it?

Reply:
yeah, probably it makes sense to remove the session and execute the full flow in both cases.
But for now we can leave it like this and improve in a couple of weeks :)